components/brave_wallet_ui/components/shared/create-network-icon/index.tsx

```diff
@@ -31,7 +31,7 @@ function CreateNetworkIcon (props: Props) {
   )
 
   const networkImageURL = stripERC20TokenImageURL(network?.iconUrls[0])
-  const isRemoteURL = isRemoteImageURL(networkImageURL)
+  const isRemoteURL = isFromDifferentOrigin(networkImageURL)
   const isDataURL = network.iconUrls[0]?.startsWith('chrome://erc-token-images/')
```

Comment:
this is also confusing because this doesn't check that it's a data URL. what is the purpose of this?